.github/workflows/release.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         uses: goreleaser/goreleaser-action@v2
         with:
           version: v0.184.0
-          args: release -f .goreleaser/linux.yml --rm-dist
+          args: release -f .goreleaser/linux.yml --rm-dist --debug
```

Comment:
might want to remove debug output in prod

Reply:
When